package.json

```
@@ -36,9 +36,12 @@
     "docs:bundle": "rm -rf docs/ && yarn allsdk docs:bundle --stream && touch docs/.nojekyll",
     "validateDocLinks": "find . -name \\*.md -not -path '*/node_modules/*' -print0 | xargs -0 -n1 yarn markdown-link-check -c ./markdownlint.json",
     "test-snippets": "yarn allsdk test-snippets --stream",
-    "test-samples": "SKIP_PREFLIGHT_CHECK=true yarn allsamples test --stream"
+    "test-samples": "SKIP_PREFLIGHT_CHECK=true yarn allsamples test --stream",
+
+    "generate-types:playground:documents": "sh scripts/codegen/build.sh"
```

```suggestion
    "generate-types:playground:documents": "scripts/codegen/build.sh"
```

(And make sure the script has +x)